### PR TITLE
API v2: TLDs

### PR DESCRIPTION
**Pull request by jodosha**

### CODE REVIEW

File: content/v2/tlds.markdown [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+
+    curl -H 'Authorization: Bearer <token>' \
+         -H 'Accept: application/json' \
+         https://api.dnsimple.com/v2/tlds/ca/extended_attributes
+
+### Response
+
+~~~json
+{
+  "data"=>[
```
Comment (weppos) [inline content/v2/tlds.markdown]: This is not JSON :smile: 

FYI, just leave the response empty, I've been reusing the fixtures we created for the API client. See 13e7832380f286ded33c636e87c43a3d866006a7

Comment (jodosha) [inline content/v2/tlds.markdown]: Ouch, forgot to replace there. :sweat_smile: Ok, I'll leave these empty.

**Comment by weppos** [comment]
LGTM :+1:

**Comment by weppos** [comment]
Merged but not deployed, I'll wait for the feature to be merged in the app. :)

**Pull request merged**